galsim/phase_psf.py

```diff
@@ -1385,7 +1394,7 @@ def __init__(self, lam_over_diam=None, lam=None, diam=None, tip=0., tilt=0., def
         optics_screen = galsim.OpticalScreen(
                 defocus=defocus, astig1=astig1, astig2=astig2, coma1=coma1, coma2=coma2,
                 trefoil1=trefoil1, trefoil2=trefoil2, spher=spher, aberrations=aberrations,
-                lam_0=lam)
+                obscuration=obscuration, annular_zernike=annular_zernike, lam_0=lam)
```

Comment:
Also, I worry the obscuration may be omitted from ```__str__```right now?  And should ```annular_zernikes``` be indicated there or is that too much detail?